packages/blade/src/components/Card/__tests__/CardInteractive.native.test.tsx

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import React from 'react';
+import { fireEvent, waitFor } from '@testing-library/react-native';
+import { Linking } from 'react-native';
+import { CardBody, Card } from '..';
+import renderWithTheme from '~utils/testing/renderWithTheme.native';
+import { Box } from '~components/Box';
+import { Text } from '~components/Typography';
+import { Button } from '~components/Button';
+
+describe('<Card />', () => {
+  it('should render a interactive Card', () => {
+    const { toJSON } = renderWithTheme(
+      <Card onClick={() => {}} scaleOnHover isSelected={true}>
+        <CardBody>Plain Card</CardBody>
+      </Card>,
+    );
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render a clickable Card', () => {
+    const { toJSON } = renderWithTheme(
+      <Card onClick={() => {}}>
+        <CardBody>Plain Card</CardBody>
+      </Card>,
+    );
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render a linkable Card', () => {
+    const { toJSON } = renderWithTheme(
+      <Card href="https://google.com">
+        <CardBody>Plain Card</CardBody>
+      </Card>,
+    );
+    expect(toJSON()).toMatchSnapshot();
+  });
+
+  it('should render a linkable Card with onClick', () => {
+    const { toJSON } = renderWithTheme(
+      <Card onClick={() => {}} href="https://google.com">
```

Comment:
why onClick and href both? what's the use case? it shall be either or right?

Reply:
<a href="" onClick={} \/> is valid and is even supported in our Link component, thus we should also support this here also, if users wants to add some analytics and such events.